src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -272,13 +284,55 @@ public static void stop() {
           if (url.getHost().contains("wikia")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS134)